response_operations_ui/static/css/components/nav.css

```diff
@@ -34,15 +42,25 @@
   border-bottom: 0.15rem solid #fff;
 }
 
-/* Tabbed navigation for secure messages */
 
-.navigation-tabs{
+
+
+
+
+
+
+
+
+/* Tabbed navigation */
+/* Using BEM         */
+
+.nav-tabs{
     list-style: none;
     padding: 0;
     border-bottom: 1px solid #666;
 }
 
-.navigation-tabs__tab, .navigation-tabs__tab--active{
+.nav-tabs__tab, .nav-tabs__tab--active{
```

Comment:
This will break the styling on the messages tabs.  Either change the name back or change the class on the page.

Reply:
works on my branch?

Reply:
The messaging css is all messed up, I'll have to have a chat with @nicprice 
Currently it's all in a separate override.css file from what nic was using but it should really mostly come from ons stylesheets and the rest should be in components. There's no rush for this PR atm but once that issue has been fixed we'll get it in